prisoner-content-hub-frontend.env

```diff
@@ -0,0 +1,24 @@
+PORT=3000
+
+# The following must match the container names in your docker-compose or local setup
+HUB_API_ENDPOINT=http://drupal:8080
+ELASTICSEARCH_ENDPOINT=http://elasticsearch:9200
+ELASTICSEARCH_INDEX_NAME='content_for_search'
+DRUPAL_DATABASE_NAME='hubdb'
+
+ENABLE_PERSONALISATION=false
+ENABLE_REDIS_CACHE=false
+ENABLE_STACK_TRACES_ON_ERROR_PAGES=true
+
+AZURE_AD_CLIENT_ID='UNSET'
+AZURE_AD_CLIENT_SECRET='UNSET'
+PRISON_API_BASE_URL='http://foo.bar'
+HMPPS_AUTH_BASE_URL='http://foo.bar'
+HMPPS_AUTH_CLIENT_ID='UNSET'
+HMPPS_AUTH_CLIENT_SECRET='UNSET'
+
+ENABLE_FEATURE_TOGGLES=true
+ENABLE_STACK_TRACES_ON_ERROR_PAGES=true
+ENABLE_MOCK_AUTH=true
+ENABLE_REDIS_CACHE='false'
+
```

Comment:
With this file, I'm not exactly sure what is in use and what isn't.  So it would be worth going through it and making sure we are only setting env variables that we actually need.

The backend will also have a separate `prisoner-content-hub-backend-local.env` file, which will be in git but also in .gitignore (so the file is created with defaults on a git checkout, buy any local amendments do not go back into the repo automatically).  We may want to copy that setup here?